### DISCUSSION TITLE
Do 'make up' run automatically when reboot?

### DISCUSSION THREAD

**Discussion by cmdntd** [alexandre-abrioux/golem-node#33] [Q&A] [answered]
I reboot and see `make up` of golem still be available. I 'm not sure if it run automatically when reboot computer, or must run again.
Could you explain this?

**Comment by alexandre-abrioux** [comment, marked as answer]
The restart option in your docker-compose is responsible for this:
https://github.com/alexandre-abrioux/golem-node/blob/709cd39f53bf2bf6cf898d990e0b79370e09ed4f/docker-compose.yml#L8

See https://docs.docker.com/compose/compose-file/compose-file-v3/#restart
